osu.Game/Online/API/Requests/Responses/APIUser.cs

```diff
@@ -201,6 +201,10 @@ private string[] playStyle
         [JsonProperty(@"playmode")]
         public string PlayMode;
 
+        [JsonProperty(@"profile_hue")]
+        [CanBeNull]
```

Comment:
```suggestion
```

not sure what this is doing here, the type of this is `int?`, i.e. a nullable value type, `[CanBeNull]` was never intended for (or did anything for) nullable value types - it's for reference types exclusively